src/component/feature/FeatureView/FeatureVariants/FeatureVariantsList/AddFeatureVariant/AddFeatureVariant.tsx

```diff
@@ -67,15 +62,17 @@ const AddVariant = ({
     const { projectId, featureId } = useParams<IFeatureViewParams>();
     const { feature } = useFeature(projectId, featureId);
     const [variants, setVariants] = useState<IFeatureVariant[]>([]);
-    const [code, setCode] = useState('');
 
-    const isJSON = () => {
+    const isJSON = (input: string): boolean => {
```

Comment:
Maybe `isValidJSON`. Alternatively, we could move this fn outside the component, make it return an error message, and call it something like `validateJSON` (just to keep it separate from React state).